src/gui/editors/PianoRoll.cpp

```diff
@@ -980,39 +980,71 @@ void PianoRoll::drawDetuningInfo( QPainter & _p, const Note * _n, int _x,
 		width() - m_whiteKeyWidth,
 		keyAreaBottom() - PR_TOP_MARGIN);
 
+	// Draw lines for the detuning automation, treating cubic hermit curves
+	// as straight lines. Also draw discrete jumps.
 	int old_x = 0;
 	int old_y = 0;
 
 	timeMap & map = _n->detuning()->automationPattern()->getTimeMap();
-	for( timeMap::ConstIterator it = map.begin(); it != map.end(); ++it )
+	for (timeMap::const_iterator it = map.begin(); it != map.end(); ++it)
 	{
-		int pos_ticks = it.key();
-		int pos_x = _x + pos_ticks * m_ppb / MidiTime::ticksPerBar();
+		// Current node values
+		int cur_ticks = POS(it);
+		int cur_x = _x + cur_ticks * m_ppb / MidiTime::ticksPerBar();
+		const float cur_level = INVAL(it);
+		int cur_y = middle_y - cur_level * m_keyLineHeight;
```

Comment:
I could change the variables I already had to rename to use camelCase, just wondering if it's worth it to do it now or leave it to the refactor since there will be a weird mix of camelCase and underscore names on the following lines.

Reply:
Tried to change the variables names there, but as expected the code got more confusing than with the current names because it started mixing both camelCase and underscore variable names. I think it's better to keep them as they are and leave the name change for a refactor of that method (where all variables can be renamed at once). Leaving comment unresolved for now so others can give an opinion if they are OK with it.